src/components/register/RegisterChallengePayment.vue

```diff
@@ -677,6 +692,20 @@ export default defineComponent({
         "
       />
     </q-banner>
+    <!-- Checkbox: With reward toggle -->
+    <div class="q-my-lg">
+      <q-checkbox
+        dense
+        v-model="isPaymentWithReward"
+        color="primary"
+        :true-value="true"
```

Comment:
According doc, default true value is `true` and false value is `false`. We don't need override default values here.